PhotoBooth/Preview Page/View Controller/PreviewViewController.swift

```diff
@@ -0,0 +1,53 @@
+import UIKit
+
+class PreviewViewController: UIViewController {
+
+    private let viewModel: PreviewViewControllerModel
+
+    lazy var previewView: PreviewView = {
+        let view = PreviewView(viewModel: viewModel.previewViewModel)
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupPreviewView()
+        previewView.backButton.addTarget(self,
+                                           action: #selector(backSelected),
+                                           for: .touchUpInside)
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        AppUtility.lockOrientation(.all)
+    }
+
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+        AppUtility.lockOrientation(.portrait)
```

Comment:
I don’t think this should be here. It works but it’s hacky because the review vc should be setting this.